config/storybook/main.ts

```diff
@@ -1,11 +1,11 @@
-import type { StorybookConfig } from '@storybook/react/types';
+import type { StorybookConfig } from '@storybook/react-vite';
 import path from 'node:path';
 import { sync as findUpSync } from 'find-up';
-import TsconfigPathsPlugin from 'tsconfig-paths-webpack-plugin';
+import { mergeConfig } from 'vite';
 
 const parent = path.resolve(__dirname, '..');
 const root = path.dirname(findUpSync('package.json', { cwd: parent }) || '.');
-const configFile = findUpSync('tsconfig.json', { cwd: root });
+const viteTsConfigPaths = require('vite-tsconfig-paths').default;
```

Comment:
No `import` possible? (move up to imports)

Reply:
what do you mean?